### PR TITLE
Added Environment section in Task def

### PR DESCRIPTION
**Pull request by git4example**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,16 @@ docker run -dit --name debugger-tcpdump -v ~/.aws/:/root/.aws/ -e BUCKET={bucket
           "protocol": "tcp",
           "containerPort": 80
         }
+      ],  
+      "environment": [
+        {
+          "name": "BUCKET",
+          "value": "databackupbucketelblogs"
```
Comment (marjamis) [inline README.md]: Could you change this to a more generic name to indicate it needs filling in?

**Comment by marjamis** [comment]
Good idea to have this in the example TD as well. I have one request to change the name so it's not so specific but beyond that should be good to merge.

File: README.md (hunk 1 above)
Comment (git4example) [reply README.md]: updated bucket name

**Comment by marjamis** [comment]
Thanks for that. Ive changed up the naming convention for this but Ill add in the new naming convention for this and will add this to the TD.

Thanks again, good suggestion to add.

**Pull request closed without merging**